core/src/test/scala/integration/kafka/api/PlaintextEndToEndAuthorizationTest.scala

```diff
@@ -19,13 +19,18 @@ package kafka.api
 import org.apache.kafka.common.config.internals.BrokerSecurityConfigs
 import org.apache.kafka.common.network.ListenerName
 import org.apache.kafka.common.security.auth._
-import org.junit.Before
+import org.junit.{Before, Test}
+import org.junit.Assert._
+import org.apache.kafka.common.errors.TopicAuthorizationException
 
 // This test case uses a separate listener for client and inter-broker communication, from
 // which we derive corresponding principals
 object PlaintextEndToEndAuthorizationTest {
+  var clientListenerName: String = ""
+  var serverListenerName: String = ""
```

Comment:
private and volatile? Also, it seems like initialising them as `None` would be better.